File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md

```diff
@@ -77,6 +77,136 @@ pptxDoc.Close()
 
 {% endtabs %}
 
+## Changing MasterSlide background
+
+The following code example demonstrates how to set the background for a master slide.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+//Creates a PowerPoint presentation.
+using(IPresentation pptxDoc = Presentation.Create())
+{
+    //Accesses the first master slide in PowerPoint file.
+    IMasterSlide masterSlide = pptxDoc.Masters[0];
+    //Retrieves the background instance.
+    IBackground background = masterSlide.Background;
+    //Sets the fill type for background as Solid fill.
+    background.Fill.FillType = FillType.Solid;
+    //Creates instance for solid Fill.
```

Comment:
@Suriya-Balamurugan - Please modify the XML comment as `Gets the instance of solid Fill.`

Reply:
Addressed.